File: backend/src/main/java/me/performancereservation/domain/refund/dto/UpdateBankInfoRequest.java
package me.performancereservation.domain.refund.dto;

import jakarta.validation.constraints.NotNull;

public record UpdateBankInfoRequest(
        @NotNull Long refundId, // 정보 추가할 환불id
        @NotNull String account, // 환불 받을 계좌번호
        @NotNull String bank, // 환불 받을 은행
        @NotNull String depositorName // 입금자명
) {
}

File: backend/src/main/java/me/performancereservation/domain/refund/dto/UpdateBankInfoRequest.kt
package me.performancereservation.domain.refund.dto

import jakarta.validation.constraints.NotNull

data class UpdateBankInfoRequest(
    @field: NotNull
    val refundId: Long,  // 정보 추가할 환불id
    @field: NotNull
    val account: String,  // 환불 받을 계좌번호
    @field: NotNull
    val bank: String,  // 환불 받을 은행
    @field: NotNull
    val depositorName: String // 입금자명
)
